### PR TITLE
fix: remove defu from module

### PR DESCRIPTION
**Pull request by btkostner**
Fixes https://github.com/moritzsternemann/vue-plausible/issues/5

Because the configuration has no nested objects, defu has the same result as just using the object spread syntax. So we can remove it and lower the bundle even more.

Order of configuration from least to most priority:
- `defaultOptions`
- `this.options.plausible`
- `moduleOptions`

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,16530 +1,8 @@
 {
   "name": "vue-plausible",
   "version": "1.1.2",
-  "lockfileVersion": 2,
```
Comment (moritzsternemann) [inline package-lock.json:4]: Seems like we might be using a different version of npm, which probably caused the change in `lockfileVersion` and the large amount of deletions.

**Review by moritzsternemann — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! We should get deletions in `package-lock.json` sorted before merging.
Removing `defu` using `npm` version 7.5.0 produced only 4 changes for me.

File: package-lock.json (hunk 1 above)
Comment (btkostner) [reply package-lock.json]: Sorry about that. I pushed an update with `npm` `7.5.4`. The line count for `package-lock.json` is down to only 20 lines. It still includes `defu` but only for development (probably for another dependency). Let me know if there is anything else :confetti_ball:

**Review by moritzsternemann — APPROVED** [review]

**Comment by moritzsternemann** [comment]
Looking good. Thanks again!

**Pull request merged**